Sources/FunctionalKit/Result.swift

```diff
@@ -5,18 +5,17 @@ import Abstract
 
 // sourcery: functor
 // sourcery: secondaryParameter = "Failure"
+// sourcery: secondaryParameterOnRight = true
 // sourcery: monad
 // sourcery: traversable
 // sourcery: testFunctor
 // sourcery: testApplicative
 // sourcery: testMonad
 // sourcery: testConstruct = "success(x)"
 // sourcery: testSecondaryParameter
-public enum Result<Failure,Parameter> where Failure: Error {
-    case success(Parameter)
-    case failure(Failure)
+extension Result {
     
-    public func run() throws -> Parameter {
+    public func run() throws -> Success {
```

Comment:
This is redundant now. Swift `Result` already includes a `get()` method that does the same thing.